src/Components/Button/Button.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { NavButton } from './Button.styles.js';
+import { StyledButton } from './Button.styles.js';
 
 export const Button = (props) => {
-    const {label,type} = props;
+    const {type,functionality,label} = props;
```

Comment:
add spacing in your destructuring `const { type, functionality, label } = props;`